packages/core-components/src/components/OAuthRequestDialog/LoginRequestListItem.tsx

```diff
@@ -59,7 +59,7 @@ const LoginRequestListItem = ({ request, busy, setBusy }: RowProps) => {
   const IconComponent = request.provider.icon;
 
   return (
-    <ListItem button disabled={busy} classes={{ root: classes.root }}>
+    <ListItem disabled={busy} classes={{ root: classes.root }}>
```

Comment:
I noticed that that entire row _does_ respond to hover, but clicking on the `ListItem` doesn't do anything, a user still needs to click on the nested "Log In" button.  

If this an an oversight, we could add an `onClick` to the `ListItem` but what action should clicking on the `ListItem` be?  If it is to be the same as `Log In` is there a need to have that nested "Button" at all?

Reply:
As a bit of historical background, this used to be just the list without the button, and the whole row clickable as usual. But users were confused, not realizing that it actually was a call to action and that the rows were clickable at all, so they closed the dialog and got a warning. So the button was added to visually clarify things, which made users happier.

Now, we're probably in a bad hybrid state as you've noted. Personally, I think you are right in your comment, it would make sense to keep only the button clickable and leave the rest of the row "passive" (doesn't have to have hover events even). The hit target is large enough and clear enough anyway.